grid_map_demos/CMakeLists.txt

```diff
@@ -54,9 +44,8 @@ catkin_package(
 ## Your package locations should be listed before other locations
 include_directories(
   include
-  ${catkin_INCLUDE_DIRS}
-  ${EIGEN3_INCLUDE_DIR}
-  ${OCTOMAP_INCLUDE_DIR}
+  # ${EIGEN3_INCLUDE_DIR}
```

Comment:
Dont you still need these?

Reply:
These includes seemed redundant, things building and running fine as is (locally at least atm). octomap already included when depending on grid_map_octomap and same for eigen as well (not sure which package off the top of my head). Will delete them in the final grid_map_demos port to confirm that nothing needs, if thats okay.